yamlgen/deploy/bottlerocket-update-operator.yaml

```diff
@@ -764,12 +764,14 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: spec.nodeName
-        - name: MAX_CONCURRENT_UPDATE
-          value: '1'
         - name: UPDATE_WINDOW_START
           value: 0:0:0
         - name: UPDATE_WINDOW_STOP
           value: 0:0:0
```

Comment:
I think it makes sense for us to unset one of these. See reasoning around comparison's to the default value.